instrumentation/apolloconfig-apolloclient-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/apolloconfig/apolloclient/v1_0/ApolloConfigInstrumentationModule.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.apolloconfig.apolloclient.v1_0;
+
+import static java.util.Collections.singletonList;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.javaagent.extension.instrumentation.InstrumentationModule;
+import io.opentelemetry.javaagent.extension.instrumentation.TypeInstrumentation;
+import java.util.List;
+
+@AutoService(InstrumentationModule.class)
+public class ApolloConfigInstrumentationModule extends InstrumentationModule {
+  public ApolloConfigInstrumentationModule() {
+    super("apolloconfig-apolloclient", "apolloconfig-apolloclient-1.0");
```

Comment:
maybe `apolloconfig-client`? I guess the package name and module name could also be adjusted accordingly.

Reply:
It seems that we don't have a unified standard

![image](https://github.com/user-attachments/assets/b6e22997-bb20-444d-adfa-1eade3374559)